### PR TITLE
Weather widget ab test

### PR DESCRIPTION
**Pull request by uplne**
AB test for weather widget

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+define([
+    'common/modules/weather',
+    'common/utils/config'
+], function (
+    weather,
+    config
+    ) {
+    return function () {
+        this.id = 'WeatherSwitch';
+        this.start = '2014-11-26';
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js]: started last year?

File: static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        this.expiry = '2015-02-01';
+        this.author = 'Steve Vadocz';
+        this.description = 'Test the accuracy of weather location';
+        this.audience = 0;
```
Comment (ironsidevsquincy) [inline static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js]: not running over any audience?

**Comment by ironsidevsquincy** [comment]
you also need to add it to https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/experiments/ab.js#L40

File: static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js (hunk 2 above)
Comment (uplne) [inline static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js]: yes, we will be just sending links first

File: static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    weather,
+    config
+    ) {
+    return function () {
+        this.id = 'WeatherSwitch';
+        this.start = '2015-01-13';
+        this.expiry = '2015-02-01';
+        this.author = 'Steve Vadocz';
+        this.description = 'Test the accuracy of weather location';
+        this.audience = 0;
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js]: Are you sure you want a 0 audience?

**Comment by phamann** [comment]
:+1: Other than comments

File: static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js (hunk 3 above)
Comment (uplne) [inline static/src/javascripts/projects/common/modules/experiments/tests/weather-component.js]: Yes, as we just want to send links around first

**Comment by gklopper** [comment]
:+1: , this is not really an AB test (if I read correctly) and I figure it would be much easier to achieve the same thing with user prefs (in the future, don't bother changing)...

**Pull request merged**